package.json

```diff
@@ -5,6 +5,7 @@
   "main": "server/server.js",
   "private": true,
   "scripts": {
+    "start": "webpack serve --mode=development --node-env=production",
```

Comment:
You can use the default `npm start` command to start development of your client in browser, w/o node server.
It's would be useful to develop the layout and CSS for the frontend.

Reply:
Definitely useful to have HMR for the front-end and not having to re-compile manually.
But with this, the custom Back-end routes are not started (which means i can't test all of my front-end).

Played around a bit and managed to (kind-of hackily) attach the custom back-end routes to webpack-dev-server.
Works good so far, but does not restart on changes to the back-end code.

The best would be if there was a solution that restarts on both back-end and front-end changes.
If you have ideas by chance, let me know.

Reply:
I have added the `npm start` command to check whether all frontend assets (CSS, JS, etc) are correct generated and  linked in HTML.

Reply:
To enable live reload after changes of frontend assets, just add the snippet into Webpack config:

```js
  devServer: {
    static: path.resolve(__dirname, 'client/dist'), // the same path as by output.path
    watchFiles: {
      paths: ['client/**/*.*'], // the watch directory
      options: {
        usePolling: true,
      },
    },
  },
```

Reply:
Yeah, figured that out already.

Just wondering how to reload both front- and back-end code automatically

Reply:
You can use the `pm2` - the manager for Node.Js: https://pm2.keymetrics.io
It can [watch & reload you backend](https://pm2.keymetrics.io/docs/usage/watch-and-restart/).